### DISCUSSION TITLE
Test current commit of action

### DISCUSSION THREAD

**Discussion by artis3n** [community/community#26418] [Actions]
<p>When I put up a PR to my Action, I want to run a workflow that test’s that commit to see whether the code changes break the action before publishing a release to the marketplace. I’m trying to do:</p>
<pre><code>uses: artis3n/ansible_galaxy_collection@${{ github.sha }}
</code></pre>
<p>However it doesn’t seem like Actions lets me specify a variable in the ‘uses’ section. Is there a way I can get the action to test on its current commit hash before merging changes and publishing a release?</p>
<p>This is the error I’m getting when trying the above:</p>
<pre><code>- Your workflow file was invalid: The pipeline is not valid. .github/workflows/main.yml (Line: 19, Col: 15): Unrecognized named-value: 'github'. Located at position 1 within expression: github.sha
</code></pre>

**Comment by artis3n** [comment, marked as answer]
<p>It turns out that</p>
<pre><code>uses: artis3n/ansible_galaxy_collection@HEAD
</code></pre>
<p>works just fine. (EDIT: it does not)</p>
<p>GitHub Actions now has support for this, with <code>uses: /</code></p>

**Comment by SeyfSV** [comment]
<p><code>uses: /</code> does not work for me, but <code>uses: ./</code> - does</p>
